provisioning/transport/amqp/tests/ProvisioningErrorDetailsAmqpTests.cs

```diff
@@ -100,10 +114,10 @@ public void GetRetryFromRejectedReturnsNullIfNoError()
         public void GetRetryAfterFromApplicationPropertiesSuccess()
         {
             int expectedRetryAfter = 42;
-            AmqpMessage amqpResponse = AmqpMessage.Create();
+            using var amqpResponse = AmqpMessage.Create();
```

Comment:
The type is good to have here. We are not doing a new so Create could be returning any other type as well.

Reply:
This was intellisense suggested, but I can change it back, sure